### PR TITLE
feat: add seminar page

### PR DESCRIPTION
**Pull request by Roshanen**
added useBoolean for useable. still wait for fixed tailwind config

### CODE REVIEW

File: src/modules/home/_sections/seminar.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          </thead>
+          <tbody>
+            {seminarData.map((entry) => (
+              <tr key={entry.time}>
+                <td className="w-1/3 border-none p-10 pl-20">{entry.time}</td>
+                <td className="w-1/3 border-none p-10">{entry.room1}</td>
+                <td className="w-1/3 border-none p-10">{entry.room2}</td>
+              </tr>
+            ))}
+          </tbody>
```
Comment (Peqchji) [inline src/modules/home/_sections/seminar.tsx:107]: ไปใช้ Tabs component ของ shadcn

Comment (Kittonn) [reply src/modules/home/_sections/seminar.tsx:107]: 🤩

**Comment by Roshanen** [comment]
- style almost finish
- SeminarTable component have table that can't render each row after dehydrate, maybe change to div+flex to minimize component
- page.tsx wait for rearrange

File: src/modules/home/_sections/seminar.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      <div className="absolute left-0 top-0 h-72 w-72 rounded-full bg-orange-200 opacity-30 blur-2xl"></div>
+
+      <div className="relative mx-auto flex h-svh w-10/12 flex-col md:w-3/4">
+        <h2 className="w-full text-3xl font-bold md:text-4xl">
+          กำหนดการสัมมนา
+        </h2>
+
+        <Tabs defaultValue="day1" className="flex flex-col gap-2">
+          <TabsList className="mx-10 my-6 flex w-fit flex-row flex-wrap gap-4 md:gap-6 lg:gap-20">
+            <TabsTrigger
```
Comment (Kittonn) [inline src/modules/home/_sections/seminar.tsx:27]: loop dai mai

File: src/shared/components/seminar-table.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import { SeminarData } from '@/shared/types/seminar-data';
+
+export function SeminarTable({ data }: { data: SeminarData[] }) {
```
Comment (Kittonn) [inline src/shared/components/seminar-table.tsx:3]: data -> seminar dai mai

File: src/shared/constants/seminar-data.ts [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    },
+    {
+      time: '12:00 AM',
+      room1: 'Tom McCoy Seminar',
+      room2: 'Carolyn Goodwin Seminar',
+    },
+  ],
+  day2: [
+    {
+      time: '09:00 AM',
```
Comment (Kittonn) [inline src/shared/constants/seminar-data.ts:33]: startAt -> timestamp (iso date)
endAt  -> timestamp (iso date)

make function format date

**Comment by Roshanen** [comment]
- will commit function convert seminar data (DB) -> seminar list (front) later

**Pull request merged**